city_scrapers/spiders/fortx_Tarrant_County_Commissioners_Court.py

```diff
@@ -0,0 +1,109 @@
+import json
```

Comment:
now that we've switched over to `data = response.json()` we should be able to remove this dependency.

Reply:
the `json` package is still being used in `scrapy.Request` to dump the dictionary with the `committeeId` as a JSON value in [line 44](https://github.com/msrezaie/city-scrapers-fortx/blob/tarrant-county-commissioners-court-spider/city_scrapers/spiders/fortx_Tarrant_County_Commissioners_Court.py#L44).